sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -800,7 +800,7 @@ class VersionsSuite extends SparkFunSuite with Logging {
       }
     }
 
-    test(s"$version: read avro file containing decimal") {
+    test(s"$version: SPARK-17920: read avro file containing decimal") {
```

Comment:
do you mean SPARK-17920 is already fixed because this test passes?

Reply:
@cloud-fan , Yes